private/buf/cmd/buf/buf_test.go

```diff
@@ -1944,19 +1944,6 @@ func TestFormatDiff(t *testing.T) {
 @@ -1,13 +1,7 @@
 -
  syntax = "proto3";
- 
```

Comment:
This test is affected if there is whitespace or not in the diff - this is kind of a mess across IDEs, just simplifying it to test what we need to